lib/Interpreter/IncrementalExecutor.cpp

```diff
@@ -121,6 +87,34 @@ std::unique_ptr<TargetMachine>
                                           OptLevel));
   return TM;
 }
+} // anonymous namespace
+
+IncrementalExecutor::IncrementalExecutor(clang::DiagnosticsEngine& diags,
+                                         const clang::CompilerInstance& CI):
+  m_externalIncrementalExecutor(nullptr)
+#if 0
```

Comment:
I didn't add it.
I've enabled its use in a subsequent commit to actually report errors properly.